const rejectedPromise = Promise.reject(new Error("エラー"));

// const rejectedPromise = new Promise((resolve, reject) => {
//   reject(new Error("エラー"));
// });
